.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will trigger tests when 
+
+name: Run Python Tests
```

Comment:
Can this be more specific? *Sanity* test or *Regression* test? instead of *Python* test

Reply:
I have changed the name to `Run Unit Tests`